build/storage/recipes/nvme/hot-plug.md

```diff
@@ -0,0 +1,146 @@
+# Hot-plug recipe
+
+This recipe describes how to perform NVMe hot-plug to a running KVM host.
+The NVMe device namespaces are backed up by an ideal targets on
+`storage-target-platform` machine exposed over NVMe/TCP.
+
+For this recipe, two physical machines are required.
+They are referred as `storage-target-platform` and `ipu-storage-container-platform`.
```

Comment:
correct all occurrences to "referred to as"

Reply:
adjusted